City-of-Helsinki/kerrokantasi-ui
# Ker 209 hds 3 header

This PR replaces the old Header component with HDS 3.0.0 Header component.
[KER-209](https://helsinkisolutionoffice.atlassian.net/browse/KER-209)

How to test:
- Check that the new Header looks OK. It should have a generic HDS style
- Try change languages
- Try to login/logout
- Try mobile screen size and open the navigation menu

[KER-209]: https://helsinkisolutionoffice.atlassian.net/browse/KER-209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
